how-to/laravel.md

```diff
@@ -0,0 +1,653 @@
+---
+title: How to use PGroonga with Laravel
+---
+
+**# 01. Introduction**
+In this guide we will walk through building a Laravel application from scratch with PGroonga. 
+
+**# 02. Installation**
```

Comment:
Can we remove explicit numbering (`02.`)?
Could you use `## ...` level heading because we need to use `# ...` level heading for the title?

```suggestion
# Installation
```